streaming/src/main/scala/org/apache/spark/streaming/StreamingContext.scala

```diff
@@ -144,6 +144,13 @@ class StreamingContext private[streaming] (
     }
   }
 
+  if (sc.conf.contains("spark.cores.max")) {
+    val totalCores = sc.conf.getInt("spark.cores.max", 1)
```

Comment:
@SOmeONee I'm not quite following your comment here. You assume that default value is "1" which changes the semantics of this configuration. What's more, checking whether "<=1" is not so useful as I mentioned above.